typescript/sdk/src/deploy/verify/utils.ts

```diff
@@ -44,3 +46,24 @@ export function getContractVerificationInput(
   const args = getConstructorArguments(contract, bytecode);
   return buildVerificationInput(name, contract.address, args, isProxy);
 }
+
+/**
+ * Check if the artifact should be added to the verification inputs.
+ * @param verificationInputs - The verification inputs for the chain.
+ * @param chain - The chain to check.
+ * @param artifact - The artifact to check.
+ * @returns
+ */
+export function shouldAddVerificationInput(
+  verificationInputs: ChainMap<ContractVerificationInput[]>,
+  chain: ChainName,
+  artifact: ContractVerificationInput,
+): boolean {
+  return !verificationInputs[chain].some(
+    (existingArtifact) =>
+      existingArtifact.name === artifact.name &&
+      existingArtifact.address === artifact.address &&
```

Comment:
could you use `eqAddress` from `@hyperlane-xyz/utils` for this
```ts
eqAddress(existingArtifact.address, artifact.address) &&
```

Reply:
tks, I'll change now

Reply:
Updated